tests/integration/test_interface.py

```diff
@@ -121,18 +126,18 @@ def test_generate_dataset_from_sample_and_source(
         DATASETS.ssa.name,
         DATASETS.tax_w2_1099.name,
         DATASETS.wic.name,
-        "TODO: tax_1040",
+        DATASETS.tax_1040.name,
     ],
 )
 def test_seed_behavior(dataset_name: str, user_config, request):
     """Tests seed behavior"""
     if "TODO" in dataset_name:
         pytest.skip(reason=dataset_name)
-    data = request.getfixturevalue(f"sample_data_{dataset_name}")
+    noising_function = DATASET_GENERATION_FUNCS.get(dataset_name)
```

Comment:
Nit: this would be clearer if it was called `generation_function`, since it's the interface's `generate_x` function.